lib/scoped.js

```diff
@@ -0,0 +1,156 @@
+module.exports = scoped
+
+var merge = require('lodash.merge')
+var EventEmitter = require('events').EventEmitter
+var toId = require('pouchdb-hoodie-api/utils/to-id')
+
+function scoped (api, type) {
+  var emitter = new EventEmitter()
+
+  var scopedApi = merge(
+    {},
+    api,
```

Comment:
Why do you merge the current API into what `store.scoped(type)` returns? Some methods don't make sense on scoped APIs, like `.connect` and `.disconnect`. `.hasLocalChanges` would be nice to have, but I don't know how we'd implement that right now. And the other methods need to be wrapped due to the type scope.